src/karts/kart.cpp

```diff
@@ -1890,6 +1900,29 @@ void Kart::handleMaterialGFX(float dt)
 {
     const Material *material = getMaterial();
 
+    if (m_skidding->isSkidding())
+    {
+        m_dirt_factor += 0.001f;
+        if (m_dirt_factor > 1.0f)
+        {
+            m_dirt_factor = 1.0f;
+        }
+        m_dirt_reset_counter = 2.0f;
+    }
+    else
+    {
+        m_dirt_reset_counter -= 0.01f;
+        if (m_dirt_reset_counter < 0.0f)
+        {
+            m_dirt_reset_counter = 0.0f;
+            m_dirt_factor -= 0.002f;
```

Comment:
This too should depend on the dt.

Besides, apart from waiting ~6s without dirtying the kart, maybe the cleaning of the kart could depend on some other things : 
- Speed
- Rescue